src/components/ChatEntry.js

```diff
@@ -3,20 +3,63 @@ import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  //display tomes as ..years ago
+  let currentYear = new Date().getFullYear();
+  let pastYear = parseInt(props.timeStamp.slice(0, 4));
+  // console.log(pastYear);
+  const agoTime = currentYear - pastYear;
+  console.log(props.sender);
+
+  //event handler for like button click
+  const handleHeartClick = () => {
+    const messageUpdated = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.heartToggling(messageUpdated);
+  };
+
+  const button = props.liked ? (
+    <button onClick={handleHeartClick} className="like">
+      ❤️
+    </button>
+  ) : (
+    <button onClick={handleHeartClick} className="like">
+      🤍
+    </button>
+  );
```

Comment:
Great work constructing the event handler and using conditional rendering to change the heart icon!